samples/wifi/sta/README.rst

```diff
@@ -129,6 +135,37 @@ Testing
       <inf> sta: ==================
       <inf> sta: State: DISCONNECTED
 
+Power management testing
+************************
+
+This sample can be used to measure current consumption of both the nRF5340 SoC and nRF7002 DK independently by using two separate Power Profiler Kits (PPK).
+The nRF5340 SoC is connected to the PPK1 and the nRF7002 DK is connected to the PPK2.
```

Comment:
This is very confusing. Both PPKs are PPK2s since this is the version of the PPK we use. I suggest we say connected to the first PPK2, and the nRF7002 **device** is connected to the second PPK2.